docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -141,44 +149,148 @@ Restrictions
+  * - Diarrhea
+    - 1.6 (1.21 - 2.02)
+    - **2.35 (2.17 - 2.54)**
+    - Yes
+  * - Measles
+    - 2.4 (1.61 - 3.48)
+    - **2.76 (2.01 - 3.78)**
+    - Yes
+  * - Lower Respiratory Infections (LRI)
+    -
+    - **1.23 (1.03 - 1.48)**
+    - Yes
+  * - Meningitis
+    -
+    - 3.2 (0.69 - 14.75)
+    - No (not significant)
+  * - Malaria
+    -
+    - 3.65 (2.23 - 5.97)
+    - No (only one study)
+
+The above relative risks for GBD 2017 should be interpreted as rate ratios for
+the incidence rates of diarrhea, measles, and LRI.
+
 Vivarium Modeling Strategy
 --------------------------
 
+We will use an **exposure model** (or **prevalence-only model** or **propensity
+model**) for a vitamin A deficiency, in which each simulant is initialized with a "propensity" for vitamin A deficiency, and the simulant's vitamin A status is determined by comparing this
+propensity to the overall VAD exposure/prevalence in the population.
+Such
+propensity/exposure models have been used in Vivarium for other risk factors and
+risk-attributable causes, such as child stunting, :ref:`child wasting/PEM
+<2017_cause_pem>`, and :ref:`iron deficiency anemia
+<2017_cause_iron_deficiency>`.
+
+In more detail, the basic strategy is to initialize each simulant with a
+propensity score distributed uniformly in [0,1], then compare this propensity
+score with the (location/age/sex/year/intervention-status)-dependent prevalence
+of vitamin A deficiency at each time step to determine whether the simulant has
+VAD during that time step. Each simulant's propensity is assigned only once, but
+the underlying prevalence distribution can change throughout the course of the
+simulation, which may result in a change in the simulant's vitamin A status.
+
+In particular, our modeling strategy will **not** explicitly use incidence or
+remission data for vitamin A deficiency, but only *prevalence* (which is the
+same as the exposure data for the VAD risk factor). The rationale for this approach is twofold:
+
+1.  We want to guarantee that the simulated baseline prevalence of vitamin A
+    deficiency matches the prevalence data from GBD, which is likely more
+    trustworthy than incidence and remission data.
+
+2.  Relative risks from the literature about the effects of vitamin A
+    supplementation or fortification on vitamin A status are best interpreted as
+    risk ratios for prevalence of vitamin A deficiency. The exposure model
+    provides a way to directly model these effect sizes in a way that preserves
+    this interpretation.
+
+.. todo::
+
+  Verify that effect sizes on VAD should actually be interpreted as described
+  above, and that the prevalence-only model is a good way to accurately
+  represent these numbers.
+
+  Explain why the prevalence-only model is a reasonable strategy, citing
+  incidence, remission, and prevalence data, as well as expert opinions about
+  VAD. (Perhaps this explanation should come later, e.g. in the Assumptions and
```

Comment:
Assumptions and limitations is right, I think.  That's where I'd look for the justification.